apps/src/labs/types.ts

```diff
@@ -85,23 +85,26 @@ export interface LevelProperties {
   // Not a complete list; add properties as needed.
   isProjectLevel?: 'true' | 'false';
   hideShareAndRemix?: 'true' | 'false';
-  levelData: LevelData;
-  video: LevelVideo;
+  levelData: LevelData | LevelDataVideo;
 }
 
 // Generic level configuration data used by labs that don't require
 // reloads between levels. Labs may define more specific fields.
-export interface LevelData {
+export interface LevelDataLevel {
   text?: string;
   validations?: Validation[];
   startSources: Source;
 }
 
-export interface LevelVideo {
+// The level data for a standalone_video level that doesn't require
+// reloads between levels.
+export interface LevelDataVideo {
```

Comment:
Could we instead make this a `VideoLevelData` that extends `LevelData`, similar to `MusicLevelData`? That way the video specific properties can live closer to the video specific components?

Reply:
Should anything belong in `LevelData` in that case?  It feels like `text`, `validations`, and `startSources` are rather specific to be in there...

Reply:
Hmm, yeah I felt like text/validations/startSources were somewhat generic, but i guess they are specific to levels with progress and projects. Maybe we should rename `LevelData` to `ProjectLevelData`? Then yeah we can keep it like you have it where `LevelData = ProjectLevelData | VideoLevelData`. Not sure what's best but I'm sure we'll refine this as we support different level types

Reply:
Done!